apps/box/src/navigation/MainTabs.navigator.tsx

```diff
@@ -97,6 +99,12 @@ const SettingsNavigator = () => {
       <SettingsStack.Screen name="Mode" component={ModeScreen} />
       <SettingsStack.Screen name="Pools" component={PoolsScreen} />
       <SettingsStack.Screen name="About" component={AboutScreen} />
+
+      <SettingsStack.Screen
+        name="Component Gallery"
+        component={ComponentGalleryScreen}
+      />
+      <SettingsStack.Screen name="Avatar Demo" component={AvatarDemoScreen} />
```

Comment:
Do we want to have a separate stack for the component gallery which they can just rip out when done. It would segregate the code better.

Reply:
done